install_config/topics/ocp_osp_dns.adoc

```diff
@@ -0,0 +1,45 @@
+==== Setting up DNS for {product-title}
+
+DNS service is an important component in the {product-title} environment. Regardless of 
+the provider of DNS, an organization is required to have certain records in 
+place to serve the various {product-title} components.
+
+WARNING: Using `/etc/hosts` is not valid, a proper DNS service must exist.
+
+Using the key secret of the DNS, you will be able to provide the information
+to the OpenShift Ansible Installer and it will automatically add A records for
+the target instances and the various {product-title} components. This process setup
+is described later when configuring the OpenShift Ansible Installer. 
+
+NOTE: Access to a DNS server is expected and implementation of a DNS is out of
```

Comment:
Remove NOTE: and maybe just say:

Access to a DNS server is expected. You can use link:https://access.redhat.com/labsinfo/dnshelper[Red Hat Labs DNS Helper] for assistance with access.